src/data/zmod/quotient.lean

```diff
@@ -63,3 +63,29 @@ def quotient_span_equiv_zmod (a : ℤ) :
   (quotient_span_nat_equiv_zmod a.nat_abs)
 
 end int
+
+namespace add_action
+
+open add_subgroup add_monoid_hom add_equiv quotient_add_group quotient function
+
+variables {α β : Type*} [add_group α] (a : α) [add_action α β] (b : β)
+
+/-- The quotient `<a>/stab(b)` is cyclic of order `minimal_period ((+ᵥ) a) b`. -/
+noncomputable def zmultiples_quotient_stabilizer_equiv :
+  zmultiples a ⧸ stabilizer (zmultiples a) b ≃+ zmod (minimal_period ((+ᵥ) a) b) :=
+(symm (of_bijective (map _ (stabilizer (zmultiples a) b)
```

Comment:
I find the proof below nearly impenetrable.

I think it's OK given that the mathematical content here is so trivial but I would be happier if it was easier to read, and didn't contain bracketing to a depth that requires `))))))`.

Reply:
Hopefully it's a bit better now :)